proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -444,6 +503,10 @@ Property-wrapper parameters cannot have an `@autoclosure` type.
 
 > **Rationale**: `@autoclosure` is unnecessary for the wrapped value, because the wrapped-value argument at the call-site will always be wrapped in a call to `init(wrappedValue:)` or `init(projectedValue:)`, which can _already_ support `@autoclosure` arguments.
 
+Function parameters cannot be wrapped without declaring a property wrapper.
```

Comment:
What does this mean?

Reply:
I meant that there isn't a way for declaring "argument wrappers" without using the property-wrapper feature. That is, we support a special `init(argumentValue:)` initializer or an `@argumentWrapper` attribute for declaring property-wrapper-like syntax just for wrapping function parameters. In fact, it was [brought up](https://forums.swift.org/t/pitch-4-se-0293-extend-property-wrappers-to-function-and-closure-parameters/44858/14) in the pitch discussion by Frederick Kellison-Linn. I hope you understand what I'm referring to. 

I don't know how to describe that well, though; suggestions are welcome.

Reply:
I don't think we need to include every suggestion brought up on the thread in the proposal (and when we do, it should go in Alternatives Considered instead of in the section about restrictions on the proposed feature). The proposal is already very bloated

Reply:
I thought it would make sense to explicitly mention that not differentiating function-parameter and property wrapping was not really a decision made by this proposal. However, now that you mention it, I think people (who have not really followed this proposal) will be able to figure it out.